src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
According to the checklist, the class implementation should not begin with an empty line. Please remove the empty line before the class declaration.